src/main/java/com/diplomaProject/StudyBe/User/Controller/UserController.java

```diff
@@ -29,9 +36,17 @@ public List<User> findAll(){
     }
 
     @GetMapping("/findByID")
-    public User findByID(@RequestBody Long id){
-        return userService.findById(id);
+    public User findByID(){
+        return userService.findById(1L);
     }
 
+    @PutMapping("/addSubject")
+    public void addSubject(@RequestBody SubjectRequestDto subjectRequest)
+    {
+        System.out.println(subjectRequest);
```

Comment:
You are not validating the data in your request dto example
`public class UserDto {
	
	private long id;
	
	// user name should not be null or empty
	// user name should have at least 2 characters
	@NotEmpty
	@Size(min = 2, message = "user name should have at least 2 characters")
	private String name;
	
	// email should be a valid email format
	// email should not be null or empty
	@NotEmpty
	@Email
	private String email;
	}`